lib/missing_validators.rb

```diff
@@ -7,6 +7,7 @@
 require 'missing_validators/validators/email_validator'
 require 'missing_validators/validators/url_validator'
 require 'missing_validators/validators/mac_address_validator'
+require 'missing_validators/validators/ip_address_validator'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.